p/i/index.php

```diff
@@ -26,7 +26,6 @@
 } else {
 	session_cache_limiter('');
 	Minz_Session::init('FreshRSS');
-	Minz_Session::_param('keepAlive', 1);	//For Persona
```

Comment:
@Alkarex I did a source search for Persona and removing this should be harmless but I'm not completely sure if it's also *useless* without Persona.

Reply:
I think we should keep it even without Persona. Let me check tonight.

Reply:
Please keep the line but change the comment to: `//To prevent the PHP session to expire`
and please link from the commit message to those two references:
https://github.com/FreshRSS/FreshRSS/commit/3d876091e1268e3ccd5036449a4deb5134936206
https://github.com/FreshRSS/FreshRSS/issues/358#issuecomment-31931484

Reply:
Kind reminder @Frenzie :-)

Reply:
I did that four days ago, lol! :-P

Reply:
Indeed! Sorry, I looked at the wrong line :P